styles/gitlab/catppuccin.user.css

```diff
@@ -0,0 +1,955 @@
+    }
+
+    .err,
+    .gd,
+    .no,
+    .ne,
+    .nv,
+    .vc,
+    .vg,
+    .vi {
+      color: @red;
+    }
+
+    .m,
+    .se,
+    .si,
+    .mf,
+    .mh,
+    .mi,
+    .mo,
+    .il,
+    .l {
+      color: @peach;
+    }
+
+    .c,
+    .cm,
+    .cp,
+    .c1,
+    .cs,
+    .cd,
+    .sd,
+    .gp {
+      color: @subtext0;
+    }
+
+    .gh,
+    .gu {
+      color: @sky;
+    }
+
+    .k,
+    .kc,
+    .kd,
+    .kp,
+    .kr {
+      color: @lavender;
+    }
+
+    .kt,
+    .nn,
+    .nc {
+      color: @yellow;
+    }
+
+    .n,
+    .nb,
+    .ni,
+    .nl,
+    .py,
+    .w,
+    .sc,
+    .bp,
+    .p {
+      color: @text;
+    }
+
+    .kn,
+    .nd,
+    .nt,
+    .ow,
+    .o {
+      color: @subtext1;
+    }
+
+    .nf,
+    .na,
+    .nx,
+    .hljs-title,
+    .hljs-title.function_ {
+      color: @blue;
+    }
+
+    .gi,
+    .sx,
+    .sr,
+    .s1,
+    .ss,
+    .ld,
+    .s,
+    .sh,
+    .s2,
+    .sb,
+    .gs {
+      color: @green;
+    }
+
+    .nb {
+      color: @peach;
+    }
+
+    .cd {
+      color: @subtext0;
+    }
+    .ci-icon {
+      &.ci-icon-variant-success {
+        background-color: if(@lookup = latte, lighten(@green, 35%), @green);
+        border-color: if(@lookup = latte, lighten(@green, 35%), @green);
+
+        .ci-icon-gl-icon-wrapper {
+          color: if(@lookup = latte, lighten(@green, 35%), @green);
+          background-color: if(@lookup = latte, @green, darken(@green, 35%));
+          border-color: if(@lookup = latte, @green, darken(@green, 35%));
+        }
+      }
+
+      &.ci-icon.ci-icon-variant-info {
+        background-color: if(@lookup = latte, lighten(@blue, 35%), @blue);
+        border-color: if(@lookup = latte, lighten(@blue, 35%), @blue);
+
+        .ci-icon-gl-icon-wrapper {
+          color: if(@lookup = latte, lighten(@blue, 35%), @blue);
+          background-color: if(@lookup = latte, @blue, darken(@blue, 35%));
+          border-color: if(@lookup = latte, @blue, darken(@blue, 35%));
+        }
+      }
+
+      &.ci-icon.ci-icon-variant-warning {
+        background-color: if(@lookup = latte, lighten(@peach, 35%), @peach);
+        border-color: if(@lookup = latte, lighten(@peach, 35%), @peach);
+
+        .ci-icon-gl-icon-wrapper {
+          color: if(@lookup = latte, lighten(@peach, 35%), @peach);
+          background-color: if(@lookup = latte, @peach, darken(@peach, 35%));
+          border-color: if(@lookup = latte, @peach, darken(@peach, 35%));
+        }
+      }
+
+      &.ci-icon.ci-icon-variant-danger {
+        background-color: if(@lookup = latte, lighten(@red, 35%), @red);
+        border-color: if(@lookup = latte, lighten(@red, 35%), @red);
+
+        .ci-icon-gl-icon-wrapper {
+          color: if(@lookup = latte, lighten(@red, 35%), @red);
+          background-color: if(@lookup = latte, @red, darken(@red, 35%));
+          border-color: if(@lookup = latte, @red, darken(@red, 35%));
+        }
+      }
```

Comment:
Might be a good place to create a mixin or similar to avoid repeating the exact same lines only with different classnames and colors.